yt/visualization/plot_window.py

```diff
@@ -182,9 +182,19 @@ class PlotWindow(ImagePlotContainer):
         The size of the window on the longest axis (in units of inches),
         including the margins but not the colorbar.
     right_handed : boolean
+        Depreceated, please use flip_horizontal.
         Whether the implicit east vector for the image generated is set to make a right
         handed coordinate system with a north vector and the normal vector, the
         direction of the 'window' into the data.
+    swap_axes: boolean
+        If True, will transpose the final image and swap axes labels. default False.
+        Note that if the swap_axes attribute is set, the behavior of all image
+        modifications (e.g., pan or set_origin) are unchanged and remain relative
+        to the underlying coordinate system and not the image display axes.
```

Comment:
For the record, this seems fine to me. However I think there is currently a slight inconsistency in that `aspect` is still going to parametrise height/width regardless of this flag's state (that is, unless I'm reading the code wrong)